bin/doh-cli

```diff
@@ -14,6 +14,13 @@ url = {
     "cloudflare":    "https://cloudflare-dns.com/dns-query",
     "quad9":         "https://dns.quad9.net/dns-query",
     "cleanbrowsing": "https://doh.cleanbrowsing.org/doh/family-filter/?ct",
+    # CIRA's Canadian Shield
+    # - "Private is a free DNS resolution service"
+    # - "Protected adds malware and phishing blocking"
+    # - "Family blocks malware and phishing plus pornographic content"
+    "cira":          "https://private.canadianshield.cira.ca/dns-query",
+    "cira-protect":  "https://protected.canadianshield.cira.ca/dns-query",
+    "cira-family":   "https://protected.canadianshield.cira.ca/dns-query",
```

Comment:
Shouldn't that url be different than the one above?

Reply:
Oh, good catch! Should be family.canadianshield.

Reply:
Fixed in 52bc372!

```
(venv) jsteel@blacktop:~/git/doh-cli$ git log --all --decorate --oneline --graph --max-count=10 # git log a dog
* 52bc372 (HEAD -> CIRA-8, yugen/CIRA-8) 8 - CIRA Support
| * 2975c8b (yugen/AnyRecordType, AnyRecordType) Allow user to use any type (IE: MX Records)
|/  
* ea8be95 (origin/master, origin/HEAD) Support custom DoH endpoints, more verbose debug
* 9c001f2 (yugen/master, master) Add author email
* bea9543 Add long description content type
* 5c5908c Initial commit

(venv) jsteel@blacktop:~/git/doh-cli$ date
Sun Mar 29 11:04:05 EDT 2020

(venv) jsteel@blacktop:~/git/doh-cli$ curl http://xtube.com --resolve xtube.com:80:$(doh-cli --dns cira xtube.com A | jq '.[0].Answer' -r) -sD /dev/stderr > /dev/null
HTTP/1.1 301 Moved Permanently
Content-length: 0
Location: https://www.xtube.com/

(venv) jsteel@blacktop:~/git/doh-cli$ curl http://xtube.com --resolve xtube.com:80:$(doh-cli --dns cira-family xtube.com A | jq '.[0].Answer' -r) -sD /dev/stderr > /dev/null
HTTP/1.1 302 Moved Temporarily
Server: awselb/2.0
Date: Sun, 29 Mar 2020 15:04:14 GMT
Content-Type: text/html
Content-Length: 126
Connection: keep-alive
Location: https://cira.ca:443/CanadianShield/Active/FamilyFilter
```